test/bundle.test.ts

```diff
@@ -40,10 +40,10 @@ describe.skipIf(isWindows)('minimal nuxt application', () => {
 
   it('default server bundle size', async () => {
     stats.server = await analyzeSizes(['**/*.mjs', '!node_modules'], serverDir)
-    expect(stats.server.totalBytes).toBeLessThan(93000)
+    expect(stats.server.totalBytes).toBeLessThan(94400)
```

Comment:
the @unhead/vue dependency is being transpiled now, previously it wasn't (it's slightly bigger than @vueuse/head was)